spec/extend-tests/subject-operator/203_test_extend_with_subject_transfers_subject_to_target.hrx

```diff
@@ -1,3 +1,8 @@
+<===> options.yml
+---
+:ignore_for:
+- libsass
```

Comment:
Same here; this is now testing a feature that no implementation supports, so it should just be deleted.